### PR TITLE
Test windows docker image.

### PR DESCRIPTION
**Pull request by godofredoc**
Do not review.

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -262,3 +262,19 @@ task:
         # TODO(jmagman): Lint macOS podspecs but skip any that fail library validation.
         - find . -name "*.podspec" | xargs grep -l "osx" | xargs rm
         - ./script/tool_runner.sh podspecs
+
+# Windows tasks
+task:
+  << : *FLUTTER_UPGRADE_TEMPLATE
+  gke_container:
+    dockerfile: .ci/DockerfileWindows
```
Comment (fkorotkov) [inline .cirrus.yml:270]: I guess you need to set platform. The default is `linux`

```suggestion
    dockerfile: .ci/DockerfileWindows
    platform: windows
```

**Comment by stuartmorgan** [comment]
Is this still needed?

**Comment by godofredoc** [comment]
Yes, I'd like to keep it. I haven't been able to run a windows task successfully.

**Comment by Hixie** [comment]
Since this hasn't been touched in many months I'm going to close it to get it off the review queue.

**Pull request closed without merging**